inc/views/base/member/activate.twig

```diff
@@ -0,0 +1,28 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.account_activation }}</title>
+{% endblock head %}
+
+{% block body %}
+    <form action="member.php" method="post">
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+            <tr>
+                <td class="thead" colspan="2"><strong>{{ lang.activate_account }}</strong></td>
+            </tr>
+            <tr>
+                <td class="trow1" width="30%"><strong>{{ lang.username }}</strong></td>
```

Comment:
Can you wrap this in a `<label for="username">` and add `id="username"` to the associated input? I've noticed a lot of inputs don't have labels linked and it's useful for accessibility to add them. If they're already there before we convert to the new theme, it makes it harder to miss them 😄